clarifai/datasets/export/inputs_annotations.py

```diff
@@ -59,18 +61,21 @@ def __init__(self,
   def _download_temp_archive(self, archive_url: str,
                              chunk_size: int = 128) -> tempfile.TemporaryFile:
     """Downloads the temp archive of InputBatches."""
-    r = self.session.get(archive_url, stream=True)
+    session = requests.Session()
+    r = session.get(archive_url, stream=True)
```

Comment:
This may need to use self.session, which is constructed with the auth headers outside this class.  I'd like a better way to set the auth headers than passing in a session, maybe using a global (or per-thread) context, so that we don't have to construct a session in the caller.  But right now I think self.session may be needed here.

Reply:
Agreed. But here the session with auth headers is giving an error for the zip archive whereas its working without auth headers. Thats why not used the session with auth headers for now..